src/api/app/models/event/service_fail.rb

```diff
@@ -21,6 +21,31 @@ def set_payload(attribs, keys)
       attribs['error'] = attribs['error'][0..800]
       super(attribs, keys)
     end
+
+    def metric_tags
+      error = case payload['error']
+              when /cannot be expanded/
+                700
```

Comment:
@bmwiedemann status code 700 & 701 are not part of http code world as far as I know, I cannot come up with an alternative quickly. Maybe we don't include those two case for now an only go with the others

Reply:
The 7xx codes are meant to signify that it does not map to a standard 4xx or 5xx code. And 500 is already used below for unclassified errors.
The idea was to add an explanation in Grafana.

Reply:
I mapped both to 409 now.